packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -1064,3 +1125,119 @@ test.serial('updateGranuleStatusToQueued() throws error if record does not exist
     }
   );
 });
+
+test.serial('updateGranuleStatusToFailed() updates granule status in the database', async (t) => {
+  const {
+    knex,
+    collectionCumulusId,
+    granule,
+    granuleId,
+    granuleModel,
+    granulePgModel,
+  } = t.context;
+  await writeGranuleFromApi({ ...granule }, knex);
+  const dynamoRecord = await granuleModel.get({ granuleId });
+  const postgresRecord = await granulePgModel.get(
```

Comment:
We should verify that the status of these records is not `failed`, so we can be sure that `updateGranuleStatusToFailed` is what is setting that status

Reply:
done.